scripts/remove-encryption-key.mjs

```
@@ -0,0 +1,244 @@
+  }
+}
+
+// Matches a full line (plus its trailing newline) shaped like:
+//   publicKey: '...anything including escaped quotes...',
+//   privateKey: "...",
+//   publicKey: `multi
+//   line template`,
+// The value quote is captured and reused to find the closing quote, and
+// [\s\S] lets the string literal span multiple lines (for template strings
+// holding an inlined PEM).
+const TS_KEY_RE = /^[ \t]*(?:publicKey|privateKey)\s*:\s*(['"`])(?:\\.|(?!\1)[\s\S])*?\1\s*,?[ \t]*\r?\n/gm
```

ESLint

[error] The quantifier '\s*' can exchange characters with '[ \t]*'. Using any string accepted by /[\t ]+/, this can be exploited to cause at least polynomial backtracking.

(regexp/no-super-linear-backtracking)